example-with-css.html

```diff
@@ -7,6 +7,13 @@
     <a href="./">&lt; Back</a>
 
     <h1>Example Recommendations Component with CSS</h1>
+    <style>
+      @media (min-width: 992px) {
+        #cru-recommendations-component-with-css {
+          width: 217px;
+        }
+      }
+    </style>
```

Comment:
How come this wasn't just added to the `example-with-css.css` file? Typically, embedded CSS goes between the `<head></head>` tags rather than in the `<body></body>`, though I'm not sure it really matters that much if this is just an example and not HTML that will be going to production.

Reply:
AEM is hosting the container this will be put it. I just didn't want it full page which isn't realistic. And it isn't really part of the CSS source I'd want on another site since it's a random number.